static/app/views/issueDetails/streamline/useEventDetailsReducer.tsx

```diff
@@ -0,0 +1,84 @@
+import {type Reducer, useCallback, useReducer} from 'react';
+
+import type {
+  SectionConfig,
+  SectionKey,
+} from 'sentry/views/issueDetails/streamline/context';
```

Comment:
is it possible to put this stuff inside this file. I don't like that i need to look in two files just to understand what is inside the EventDetailsState